src/__tests__/lights-out.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Light } from '../components/light.js';
+import { render, cleanup, fireEvent } from '@testing-library/react';
+import { BoardContainer } from '../containers/board-container.js';
+import '@testing-library/jest-dom';
+
+afterEach(cleanup);
+
+it('should start off with the correct board', () => {
+    const { getByTestId, asFragment } = render(
```

Comment:
Do you need this line or does this snapshot test work as expected with just the `expect(asFragment(..))` bit?